packages/generator/templates/app/README.md

```diff
@@ -147,7 +147,7 @@ These files are:
 
 - `blitz.config.js` is for advanced custom configuration of Blitz. It extends [`next.config.js`](https://nextjs.org/docs/api-reference/next.config.js/introduction).
 
-- `jest.config.js` contains config for Jest tests. You can [customize it if needed](https://jestjs.io/docs/en/configuration).
+- `jest.config.ts` contains config for Jest tests. You can [customize it if needed](https://jestjs.io/docs/en/configuration).
```

Comment:
```suggestion
- `jest.config.js` contains config for Jest tests. You can [customize it if needed](https://jestjs.io/docs/en/configuration).
```
keep these as js since that's the base case. Later we can optimize to set correctly based on js/ts.